openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -133,7 +133,7 @@ <h3 translate>Election result</h3>
         index="$index" heading="{{ 'Ballot' | translate }} {{ $index + 1 }}">
       <div ng-controller="AssignmentPollDetailCtrl">
         <!-- action buttons -->
-        <div class="pull-right" os-perms="assignments.can_manage">
+        <div class="pull-right" os-perms="assignments.can_manage" ng-show="assignment.phase != 2">
```

Comment:
I tested it: lodash function in template does not work. And I don't like the very long form (via 'display_name'). We already use the values  (0,1,2) in all code parts of this detail template an in pdf and projector. So: won't fix.